udp.go

```diff
@@ -0,0 +1,146 @@
+package frontman
+
+import (
+	"bytes"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/cloudradar-monitoring/frontman/pkg/iax"
+	"github.com/cloudradar-monitoring/frontman/pkg/utils"
+)
+
+func (fm *Frontman) runUDPCheck(addr *net.UDPAddr, hostname string, service string) (MeasurementsMap, error) {
+	if addr.Port <= 0 {
+		return nil, fmt.Errorf("invalid port value: %d", addr.Port)
+	}
+
+	prefix := fmt.Sprintf("net.udp.%s.%d.", service, addr.Port)
+
+	// Initialise MeasurementsMap
+	m := MeasurementsMap{
+		prefix + "success": 0,
+	}
+
+	// Start measuring execution time
+	started := time.Now()
+	// Calculate execution time in the end
+	defer func() {
+		m[prefix+"connectTime_s"] = time.Since(started).Seconds()
```

Comment:
it's called `connectTime_s` but it measures the whole connect-request-response instead. In the webcheck we call it `totalTimeSpent_s `: https://github.com/cloudradar-monitoring/frontman/blob/ec5efc63606ef4a66bfa0b92cede5096b3832361/webcheck.go#L138

Reply:
I've updated both the TCP and UDP implementation to contain connectTime_s  and totalTimeSpent_s  fields.